e2e/evm/test.go

```diff
@@ -108,7 +108,7 @@ func (s *IntegrationTestSuite) TestDeposit() {
 	}
 	//wait for vote log event
 	proposalEvent := "ProposalEvent(uint8,uint64,uint8,bytes32,bytes32)"
-	evts, err := s.client2.FetchEventLogs(context.Background(), s.bridgeAddr, proposalEvent, b, ba)
+	evts, _ := s.client2.FetchEventLogs(context.Background(), s.bridgeAddr, proposalEvent, b, ba)
```

Comment:
why is error being ignored here?

I see that within the `FilterLogs` the error could occur and would get returned back to the calling code, `FetchEventLogs`, then returned back here but the error is being ignored

Reply:
I guess it may not matter as much because this is just a test

Reply:
I answered a minute to late :smile: